bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -83,9 +83,9 @@ spec:
       - description: Metrics defines the metrics configuration for operands.
         displayName: Metrics Config
         path: observability.metrics
-      - description: EnableMetrics specifies if ServiceMonitors should be created
-          for the OpenTelemetry Collector. The operator.observability.prometheus feature
-          gate must be enabled to use this feature.
+      - description: EnableMetrics specifies if ServiceMonitor should be created for
```

Comment:
```suggestion
      - description: EnableMetrics specifies if ServiceMonitors should be created for
```

Reply:
I modified this because only one ServiceMonitor will be created. I'm not sure if it should be reverted back to ServiceMonitors or not.

Reply:
ah i see that now. in that case we should say "a ServiceMonitor"